### PR TITLE
Docker run fix

### PR DESCRIPTION
**Pull request by EdDev**

### CODE REVIEW

File: automation/run-tests.sh [hunk 1]
```diff
@@ -307,7 +307,7 @@ dump_network_info
 
 pyclean
 if [[ "$CI" != "true" ]];then
-    container_exec "cp -rf $CONTAINER_WORKSPACE /root/nmstate-workspace"
+    container_exec "cp -rf $CONTAINER_WORKSPACE /root/nmstate-workspace || true"
```
Comment (tyll) [inline automation/run-tests.sh]: This does not seem to be correct. If copying failed, the test should not continue.

Comment (EdDev) [reply automation/run-tests.sh]: I explained this in the commit. The copy may fail even if the files copy fully. And if there is a problem, the tests will fail later on.

Comment (tyll) [reply automation/run-tests.sh]: can you maybe change this to do this only if `CONTAINER_CMD == docker`?

Comment (EdDev) [reply automation/run-tests.sh]: Sure, thanks.

Comment (EdDev) [reply automation/run-tests.sh]: Will this suffice?
```bash
    if [ "$CONTAINER_CMD" == "docker" ];then
      container_exec 'rm -rf $CONTAINER_WORKSPACE/*nmstate*.rpm' || true
    else
      container_exec 'rm -rf $CONTAINER_WORKSPACE/*nmstate*.rpm'
    fi
```

Comment (tyll) [reply automation/run-tests.sh]: works for me

Comment (EdDev) [reply automation/run-tests.sh]: Done

File: automation/run-tests.sh [hunk 2]
```diff
@@ -34,8 +34,12 @@ test -t 1 && USE_TTY="-t"
 function remove_container {
     res=$?
     [ "$res" -ne 0 ] && echo "*** ERROR: $res"
-    container_exec 'rm -rf $CONTAINER_WORKSPACE/*nmstate*.rpm'
-    ${CONTAINER_CMD} rm $CONTAINER_ID -f
+    if [ "$CONTAINER_CMD" == "docker" ];then
```
Comment (tyll) [inline automation/run-tests.sh]: `if $CONTAINER_CMD --version | grep -qv podman; then` should run this only if it is not podman. With podman-docker installed, the docker command will actually be podman but show this in the version.

Comment (EdDev) [reply automation/run-tests.sh]: Done, thanks.

File: automation/run-tests.sh (hunk 1 above)
Comment (tyll) [reply automation/run-tests.sh]: This conversation was actually for the cp not the rm

Comment (EdDev) [reply automation/run-tests.sh]: Ohh, right. Fixed.

**Comment by EdDev** [comment]
CI failures on unrelated issues (OVS related).
The job that tracks NM 1.22-git passes.
Locally it passes.
Force merging this PR.

**Pull request merged**